src/notation/view/widgets/editstyle.cpp

```diff
@@ -1053,6 +1068,51 @@ EditStylePage EditStyle::pageForElement(Element* e)
     }
 }
 
+//--------------------------------------------------------
+//   arrayToString
+//--------------------------------------------------------
+
+std::string EditStyle::arrayToString(int* arr)
+{
+   std::string s;
+   for (int i=0; i<numberOfPage; i++) {
+       s = s.append(std::to_string(arr[i]).append(","));
+   }
+   return s;
+}
+
+//--------------------------------------------------------
+//   stringToArray
+//--------------------------------------------------------
+
+void EditStyle::stringToArray(std::string s, int* arr)
+{
+   int j=0;
+   std::string n = "";
+   for (int i=0; i<static_cast<int>(s.length());i++) {
```

Comment:
wha not usine `size_t`here? No cast needed then

Reply:
qtcreator gives warning "comparison of integers of different signs" ... off course both of them will always be small positive integers, and it still work without the cast.

Reply:
`length()`gives an unsigned long int`, or `size_t``, so better use that for the indices too

Reply:
should be good now ... plus fixed the indentations ;)